manifests/pcap-api.yml

```diff
@@ -78,36 +114,67 @@ update:
 
 stemcells:
   - alias: default
-    os: ubuntu-bionic
+    os: ubuntu-jammy
     version: latest
 
 releases:
   - name: bpm
-    version: 1.1.8
-    url: https://bosh.io/d/github.com/cloudfoundry/bpm-release?v=1.1.8
-    sha1: c956394fce7e74f741e4ae8c256b480904ad5942
+    url: https://bosh.io/d/github.com/cloudfoundry/bpm-release?v=1.1.21
+    version: 1.1.21
+    sha1: e8abe19ec186962828de843f8f281cddb6141904
   - name: pcap
-    version: 0+dev.7
-    url: https://github.com/domdom82/pcap-server-release/releases/download/v0.0.1/pcap-server-0.0.1.tgz
-    sha1: a1655c1992b911fef3ad0b46c5abb8a0217ef0a2
+    version: 0+dev.1682596802
   - name: bosh-dns-aliases
     version: 0.0.4
     url: https://bosh.io/d/github.com/cloudfoundry/bosh-dns-aliases-release?v=0.0.4
     sha1: 55b3dced813ff9ed92a05cda02156e4b5604b273
   - name: routing
-    version: 0.232.0
-    url: https://bosh.io/d/github.com/cloudfoundry/routing-release?v=0.232.0
-    sha1: 1150a7c89a04db07145c906ccdbfda7424e6dfc6
+    url: https://bosh.io/d/github.com/cloudfoundry/routing-release?v=0.259.0
+    version: 0.259.0
 
 variables:
+  - name: pcap_ca
+    type: certificate
+    options:
+      common_name: pcap_ca
+      is_ca: true
   - name: pcap_api_mtls
     type: certificate
-    update_mode: converge
     options:
-      ca: ((pcap_api.ca_path))
+      ca: pcap_ca
       common_name: pcap_api_mtls
       alternative_names:
         - pcap-api.service.cf.internal
       extended_key_usage:
         - client_auth
         - server_auth
+  - name: pcap_agent_mtls
+    type: certificate
+    update_mode: converge
+    options:
+      ca: pcap_ca
+      common_name: pcap_agent_mtls
+      alternative_names:
+        - pcap-agent.service.cf.internal
+      extended_key_usage:
+        - client_auth
+        - server_auth
+  - name: pcap_api_bosh_mtls
+    type: certificate
+    options:
+      ca: /bootstrap-bosh-aws-cfn07/bosh/default_ca
+      common_name: pcap_api_bosh_mtls
+      alternative_names:
+        - pcap-api.service.cf.internal
+      extended_key_usage:
+        - client_auth
+#  - name: pcap_api_platform_mtls
```

Comment:
If it does not needed, remove these lines at all.

Reply:
I left these in intentionally for the followup where we enable platform TLS

Reply:
Removed, see https://github.com/cloudfoundry/pcap-release/pull/23/commits/f4553fbb7302ad917ff8e4b8413eb93c7c830fe6